pyfr/solvers/euler/elements.py

```diff
@@ -49,6 +50,78 @@ def con_to_pri(cons, cfg):
 
         return [rho] + vs + [p]
 
+    @staticmethod
+    def validate_formulation(form, intg, cfg):
+        if form not in BaseFluidElements.formulations:
```

Comment:
Not sure if we need this? (As Euler and NS work for both.)

Reply:
Just adding the check in there in case any other formulations get added, I can remove if necessary

Reply:
Yes, I'd remove self.formulations entirely as a member (here and in AC and switch everything over to the new function based checks).

Reply:
Okay I think the plugins will need some changes

Reply:
I think plugins are okay, they can keep their current approach since they do not need more complex logic (unlike systems where things are cfg dependent).